pom.xml

```diff
@@ -201,7 +201,7 @@
                     </from>
                     <to>
                         //Once the pipeline is moved to shared services need to update this to image reference from shared services
-                        <image>169942020521.dkr.ecr.eu-west-1.amazonaws.com/local/overseas-entities-api:latest</image>
+                        <image>169942020521.dkr.ecr.eu-west-2.amazonaws.com/local/overseas-entities-api:latest</image>
```

Comment:
I think the comment is not appropriate, It should be removed. 
Do we need local in image path

Reply:
Comment is still needed, this has not been updated to the shared services image reference yet

Reply:
thanks Dholoram, think we need this comment until it's moved to shared services - will add a note to the ticket to come back and remove